Source/WebKit/UIProcess/API/Cocoa/WKWebView.mm

```diff
@@ -5212,8 +5212,7 @@ - (void)_setViewScale:(CGFloat)viewScale
     _impl->setViewScale(viewScale);
 #else
     if (_page->mainFramePluginOverridesViewScale()) {
-        if (_page->layoutSizeScaleFactorFromClient() != 1)
-            _page->setViewportConfigurationViewLayoutSize(_page->viewLayoutSize(), 1, _page->minimumEffectiveDeviceWidth());
+        _page->resetViewportConfigurationForPDFPluginIfNeeded();
```

Comment:
This line might not compile on watchOS

Reply:
I.e. you might want to wrap it around ENABLE(PDF_PLUGIN)

Reply:
Good catch, thanks! Fixed.